assets/css/customs.css

```diff
@@ -384,14 +468,19 @@ div.erp-custom-menu-container.--show-secondary .-secondary {
 }
 
 .wperp-filter-dropdown .wperp-btn.btn--cancel:hover {
-    background-color: #e9e9e9 !important;
-    color: #000 !important;
+    background-color: #eeeeee !important;
+    color: rgb(88, 88, 88) !important;
```

Comment:
The `rgb` color codes are generated by the code editor actually and for now, as they behave exactly same as the hexadecimal format, it will be waste of time to refactor them.
However, I appreciate your suggestion and the standard can be followed in future.

Reply:
Yes. That would be good.